### PR TITLE
Update to Daml Enterprise 2.7.4 [PLEN-504] 

### PR DESCRIPTION
**Pull request by davidd-da**
* Canton: rework bootstrap script
* HTTP JSON API service: use configuration file (instead of args/flags)
* Trigger service: fix log level + encoding (JSON) using environment variables
* Remove navigator as the required container image (`daml-sdk`) is no longer available

### CODE REVIEW

File: scripts/send-json-api-requests.sh [hunk 1]
```diff
@@ -37,10 +37,5 @@ do
     getParty
   fi
 
-  # Generate traffic on the health check endpoints
```
Comment (miklos-da) [inline scripts/send-json-api-requests.sh:40]: It makes sense to generate traffic on the health endpoints as well, but not necessarily in every iteration.

File: scripts/send-trigger-requests.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ do
   # Generate a trigger request
   http_code=$(curl -fsSL -o /dev/null -w "%{http_code}" -X POST \
     -H 'Content-Type: application/json' \
-    --data '{"triggerName":"e0326da3de4b3d4ef4d193907afb82bf9afb938daccea445cbca747bb79c9139:NoOp:noOp","party":"alice","applicationId": "test-app-id"}' \
+    --data '{"triggerName":"e0326da3de4b3d4ef4d193907afb82bf9afb938daccea445cbca747bb79c9139:NoOp:noOp","party":"observability","applicationId": "test-app-id"}' \
```
Comment (miklos-da) [inline scripts/send-trigger-requests.sh:15]: Nit: you might want to name this party in a more specific way, e.g., `observabilityExample` or similar.

File: canton/bootstrap.canton [hunk 3]
```diff
@@ -1,89 +1,26 @@
...
+  node.health.wait_for_running()
 }
 
-val baseDir = os.Path("data", base=os.pwd)
-val dars = os.walk(baseDir).filter(x => x.ext == "dar").sorted
-dars.foreach { dar =>
-    logger.info(s"$logPrefix Uploading DAR ${dar}")
-    myparticipant.dars.upload(dar.toString)
+// Connect participant to domain if not already done
+if(!participant1.domains.list_registered().map(_._1.domain.unwrap).contains("mydomain")) {
```
Comment (miklos-da) [inline canton/bootstrap.canton:12]:
```suggestion
if (!participant1.domains.list_registered().map(_._1.domain.unwrap).contains("mydomain")) {
```

File: scripts/send-trigger-requests.sh (hunk 2 above)
Comment (davidd-da) [reply scripts/send-trigger-requests.sh]: Renamed party to `observabilityExample`

Comment (miklos-da) [reply scripts/send-trigger-requests.sh]: You're right — sorry, forgot about that.

**Review by miklos-da — APPROVED** [review]

**Pull request merged**